NextcloudTalk/TextFieldTableViewCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+// SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+// SPDX-License-Identifier: GPL-3.0-or-later
+//
+
+import UIKit
+
+let textFieldCellIdentifier = "textFieldCellIdentifier"
```

Comment:
This should be better inside the class, as it's done in `TextViewTableViewCell`? 🤔